apps/bare-expo/ios/Podfile

```diff
@@ -24,6 +24,10 @@ abstract_target 'BareExpoMain' do
     includeTests: true,
   )
 
+  pre_install do |installer|
+    expo_pre_install!(installer)
+  end
```

Comment:
requiring the template change is not ideal, especially that's a temporary workflow. we had a cocoapods monkey patching code in https://github.com/expo/expo/tree/main/packages/expo-modules-autolinking/scripts/ios/cocoapods
any change to do the pre_install hook in the hook?

Reply:
I've now implemented the logic in a class called `installer.rb` as you suggested - this seems to work fine so I've remove the changes to our pod files and templates! Great suggestion!